napalm/eos/eos.py

```diff
@@ -159,7 +161,10 @@ def _load_config(self, filename=None, config=None, replace=True):
             commands.append(line)
 
         try:
-            self.device.run_commands(commands)
+            if self.eos_autoComplete:
```

Comment:
This should probably be `if self.eos_autoComplete is not None` in case `False` has to be enforced in the future.